class CompletedException(Exception):
    """Indicates that a task has been completed."""
    pass
